src/Sylius/Behat/Context/Api/Admin/ManagingZonesContext.php

```diff
@@ -0,0 +1,486 @@
+            'Zone could not be created'
+        );
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyEdited(): void
+    {
+        Assert::true(
+            $this->responseChecker->isUpdateSuccessful($this->client->getLastResponse()),
+            'Zone could not be edited'
+        );
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully deleted
+     * @Then I should be notified that they have been successfully deleted
+     */
+    public function iShouldBeNotifiedThatItHasBeenSuccessfullyDeleted(): void
+    {
+        Assert::true($this->responseChecker->isDeletionSuccessful(
+            $this->client->getLastResponse()),
+            'Zone could not be deleted'
+        );
+    }
+
+    /**
+     * @Then I should be notified that this zone cannot be deleted
+     */
+    public function iShouldBeNotifiedThatThisZoneCannotBeDeleted(): void
+    {
+        Assert::false(
+            $this->responseChecker->isDeletionSuccessful($this->client->getLastResponse()),
+            'Zone can be deleted, but it should not'
+        );
+    }
+
+    /**
+     * @Then I should be notified that zone with this code already exists
+     */
+    public function iShouldBeNotifiedThatZoneWithThisCodeAlreadyExists(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            'code: Zone code must be unique.'
+        );
+    }
+
+    /**
+     * @Then /^I should be notified that (code|name) is required$/
+     */
+    public function iShouldBeNotifiedThatIsRequired(string $element): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            sprintf('Please enter zone %s.', $element)
+        );
+    }
+
+    /**
+     * @Then I should be notified that at least one zone member is required
+     */
+    public function iShouldBeNotifiedThatAtLeastOneZoneMemberIsRequired(): void
+    {
+        Assert::contains(
+            $this->responseChecker->getError($this->client->getLastResponse()),
+            'members: Please add at least 1 zone member.'
+        );
+    }
+
+    /**
+     * @param CountryInterface|ZoneInterface|ProvinceInterface $objectToRemove
+     */
+    private function removeZoneMember($objectToRemove): void
+    {
+        /** @var ZoneInterface $zone */
+        $zone = $this->sharedStorage->get('zone');
+        $iri = $this->iriConverter->getItemIriFromResourceClass(ZoneMember::class, ['code' => $objectToRemove->getCode()]);
+
+        $members = $this->responseChecker->getValue($this->client->show($zone->getCode()), 'members');
+        foreach ($members as $key => $member) {
+            if ($iri === $member) {
+                $this->client->removeSubResource('members', $member);
+            }
+        }
```

Comment:
I wonder if this could not be simplified to
```suggestion
        $this->client->removeSubResource('members', $member);
```